test/libyul/yulOptimizerTests/commonSubexpressionEliminator/long_literals_as_builtin_args.yul

```diff
@@ -0,0 +1,32 @@
+object "AccessControlDefaultAdminRules4233_14" {
+    code {
+        let programSize := datasize("AccessControlDefaultAdminRules4233_14")
+        let i := mload(64)
+        // argument of datasize is a literal of length >32 which shares a common substring
+        // of length 32 with programSize, this should not be substituted
+        let j := datasize("AccessControlDefaultAdminRules4233_14_deployed")
+        codecopy(i, dataoffset("AccessControlDefaultAdminRules4233_14_deployed"), j)
+    }
+    object "AccessControlDefaultAdminRules4233_14_deployed" {
+        code {
+            {
+                revert(0, 0)
+            }
+        }
```

Comment:
You don't really need any code here for the example to make sense:
```suggestion
        code {}
```

Reply:
`soltest` is failing because CSE will get rid of this anyway. Just leave the test as is, but update the expectations properly.